pkg/process/model/agent.pb.go

```diff
@@ -1,6 +1,5 @@
-// Code generated by protoc-gen-gogo.
```

Comment:
I know this is a bit tedious, but mind if we regenerate this proto with our existing gogo version to exclude all these (hopefully) no-op changes? Should clean up the PR a little bit and make it easier to review